@navikt/core/react/src/button/button.stories.tsx

```diff
@@ -187,6 +187,6 @@ export const Chromatic: Story = {
 export const ChromaticForcedColors: Story = {
   render: Chromatic.render,
   parameters: {
-    chromatic: { disable: false, forcedColors: "active" },
+    chromatic: { disable: false },
```

Comment:
```suggestion
    chromatic: { disable: false, forcedColors: "active" },
```
They say they have fixed it, so trying to revert.

Reply:
Seems to be fixed now 🙌

Reply:
nice! (Chromatic was amazingly fast! felt like they did this fix just for us! 🙌 )

Reply:
Yeah, probably not many people using forcedColors 😅